### DISCUSSION TITLE
CLI output

### DISCUSSION THREAD

**Discussion by petarvujovic98** [near/near-sdk-js#227] [Ideas]
I think it would be good to have more concise and colorized output when running the CLI to build the contracts. Something like [colors](https://npmjs.com/package/colors) or similar.

Maybe adding some options like _verbose_ output and show only errors by default.

Just a thought...

**Comment by volovyks** [comment]
I agree that we can make it more compact. Verbose pattern + colors should work just fine.
Building steps take a lot of time, I would retain their names so users can see the progress and understand what is happening. Ideally with blinking dots or something.

The same goes for the post-install script, it prints a ton of text now.

**Reply by petarvujovic98** [reply]
My points exactly. I'll have a look at making a PR for this

**Comment by petarvujovic98** [comment]
Posting this link here as it looks like a nice library we could use for this [signale](https://github.com/klaussinani/signale).
